05 - Flex Panel Gallery/index-START.html

```diff
@@ -71,8 +91,8 @@
 
     .panel.open {
       font-size: 40px;
+      flex: 5;
```

Comment:
what does flex 5 mean?

Reply:
flex-grow specifies how much remaining space the element is going to take based on the value we give it. the higher the value the higher the width it will get. we have 5 panels with flex-grow 1. therefore, each active panel will occupy 5 time factor the remaining space. we could have taken any other value (2-3-4-6 etc...). This is just a design choice from Wes.

Reply:
❤️